BTSPocket/Resources/Extensions/UIImage+extension.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  UIImage+extension.swift
+//  BTSPocket
+//
+//  Created by bts on 06/04/21.
+//
+
+import Foundation
+import UIKit
+import Alamofire
+
+extension UIImageView {
+    
+    public func loadImage(url: URL) {
+        DispatchQueue.global().async { [weak self] in
+            if let data = try? Data(contentsOf: url) {
+                if let image = UIImage(data: data) {
+                    DispatchQueue.main.async {
+                        self?.image = image
+                    }
+                }
+            }
+        }
+    }
+    
+    public func loadProfileImage(urlString: String) {
+        let url = URL(string: urlString)
+        guard let requestURL = url else { fatalError("URl not valid") }
+        var request: URLRequest = URLRequest(url: requestURL)
+        request.httpMethod = "GET"
+        request.setValue("https://platform.bluetrail.software/", forHTTPHeaderField: "Referer")
```

Comment:
put server address in constants,  If this addresss changes, you would have to modify the code in several files instead of centralized